app/assets/stylesheets/_orders.scss

```diff
@@ -3,3 +3,54 @@
   margin:0 auto;
   margin-bottom: 30px;
 }
+
+.admin-order-title{
+  text-align:center;
+  margin-bottom:30px;
+}
+
+.admin-order-show{
+  width:70%;
+  margin:0 auto;
+  border:1px solid white;
+  background:rgba(255, 255, 255, 0.91);
+
+  tfoot{
+    td{
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Opening curly brace `{` should be preceded by one space